charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 35.5.1
+version: 35.6.0
```

Appears to be a bugfix release.

```suggestion
version: 35.6.1
```